projects/ngx-charts-on-fhir/src/lib/data-layer-list/annotation-options/annotation-options.component.spec.ts

```diff
@@ -47,4 +55,57 @@ describe('AnnotationOptionsComponent', () => {
   it('should create', () => {
     expect(component).toBeTruthy();
   });
+
+  it('called private updateModel method', () => {
+    let spyon = spyOn<any>(component, 'updateModel');
+    let formValue: any = { color: '#377eb8', label: 'Systolic Blood Pressure Reference Range', yMax: 130, yMin: 90 };
+    component['updateModel'](formValue);
```

Comment:
1. Please don't bypass `private` access modifiers in unit tests. Unit tests should be testing the public API of the component.
2. This test never executes any real code. It creates a spy, calls the spy, and verifies that that the spy was called.